apps/web/utils/reply-tracker/enable.ts

```diff
@@ -110,8 +110,11 @@ export async function enableReplyTracker({
               bcc: null,
               webhookUrl: null,
             },
+            delayInMinutes: null,
           },
-          ...(addDigest ? [{ type: ActionType.DIGEST }] : []),
+          ...(addDigest
+            ? [{ type: ActionType.DIGEST, delayInMinutes: null }]
```

Comment:
do we really need to set null here? all the other fields aren't set to null explicitly, so is this needed?